### PR TITLE
fats support for streaming runtime testing

### LINKED ISSUE #214: Add fats support for streaming runtime
FATS coverage is required to release streaming runtime with riff ~~v0.5.0~~ 
Feb 6: bumped to v0.6.0.

### PR DESCRIPTION
**Pull request by sbawaska**
closes #214

### CODE REVIEW

File: functions/repeater/java/src/main/java/io/projectriffsamples/Repeater.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import reactor.core.publisher.Flux;
+import reactor.util.function.Tuple2;
+
+import java.util.Collections;
+import java.util.function.Function;
+
+/**
+ * Accepts a Flux of String and Flux of integer and returns a flux of string
+ * that represents the string repeated integer number of times in a list.
```
Comment (ericbottard) [inline functions/repeater/java/src/main/java/io/projectriffsamples/Repeater.java]: "in a list" is a bit misleading here

Comment (ericbottard) [reply functions/repeater/java/src/main/java/io/projectriffsamples/Repeater.java]: hmmm, actually, sorry this IS what the function is doing. But do we want that?

File: macros/streaming-prereq-install.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm install --name my-kafka bitnami/kafka
+
+cat <<EOF | kubectl create -f -
```
Comment (ericbottard) [inline macros/streaming-prereq-install.sh]: This can be created by the riff CLI now

File: macros/helm-hack.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (scothis) [inline macros/helm-hack.sh]: rm

File: macros/streaming-prereq-install.sh (hunk 2 above)
Comment (scothis) [reply macros/streaming-prereq-install.sh]:
```
riff streaming kafka-provider create franz --bootstrap-servers my-kafka:9092
```

File: macros/streaming-prereq-install.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (scothis) [inline macros/streaming-prereq-install.sh]: remove from the macros dir until it's proven to be reusable

**Comment by sbawaska** [comment]
closing in favor of: https://github.com/projectriff/fats/pull/244

**Pull request closed without merging**